src/docs/user/references/Variables.adoc

```diff
@@ -307,11 +307,11 @@ Here is the list of ProActive system variables:
 | String
 
 | `PA_CLOUD_AUTOMATION_REST_URL`
-| The URL of cloud automation REST api.
+| The URL of service automation REST api.
```

Comment:
I see different names: service automation, service Automation and Service Automation
We should choose one name in the whole doc. I suggest to use **Service Automation**